app/mailers/candidate_mailer.rb

```diff
@@ -91,6 +91,19 @@ def new_offer_decisions_pending(application_choice)
     new_offer(application_choice, :decisions_pending)
   end
 
+  def chase_candidate_decision(application_choices)
+    @application_form = application_choices.first.application_form
+    @dbd_date = application_choices.first.decline_by_default_at.to_s(:govuk_date).strip
+    @application_choices = application_choices
+
+    email_address = application_choices.first.application_form.candidate.email_address
+    subject_pluralisation = application_choices.count > 1 ? 'plural' : 'singular'
```

Comment:
There's a Rails method, `pluralize`, which can help to avoid the need for this and two subjects in the localisation.

See https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize.

Reply:
Yes I'm aware of that, but since its a subject referenced in a translation I'm not sure how much if it would be any better?
``` I18n.t!("chase_candidate_decision_email.subject", application: pluralize(@application_choices.count, 'application'))``` I'll try to replace to this 👍

Reply:
I won't as discussed with @chubberlisk